src/library/plugin/pluginType.ts

```diff
@@ -17,60 +18,38 @@ export type PluginType = {
 export interface IMinAuthPlugin<PublicInputsArgs, Output> {
```

Comment:
Maybe rename PublicInputArgs -> PublicInput ?

Reply:
Nope, the idea is to provide a mechanism for the plugin to validate the public inputs used to generate the proof. (Since the prover is run on the client side, a malicious actor can mangle the public inputs however they want.) You can imagine a function `PublicInputsArgs -> PublicInputs` being called in the body of `verifyAndGetOutput`, and by doing so we can always make sure to reject those malicious proofs.

Reply:
So you're saying that a client can build a proof and then what? Set public input field to whatever they want? and this will not affect the verification process?